src/java/src/main/java/ai/onnxruntime/genai/Tensor.java

```diff
@@ -5,12 +5,16 @@
 package ai.onnxruntime.genai;
 
 import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
 
 public final class Tensor implements AutoCloseable {
-  private long nativeHandle = -1;
+  private long nativeHandle = 0;
```

Comment:
not too sure about this change, but close() checks if it's 0.

Reply:
I would expect `close()` is only called after successful construction of the object, and nativeHandle is set to the return value from createTensor if that occurs.